compiler/noirc_evaluator/src/ssa/ssa_gen/program.rs

```diff
@@ -101,13 +107,39 @@ impl Ssa {
 
 impl Display for Ssa {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if self.globals.dfg.values_iter().len() > 0 {
+            write!(f, "{}", self.globals)?;
+        }
+
         for function in self.functions.values() {
             writeln!(f, "{function}")?;
         }
         Ok(())
     }
 }
 
+impl Display for GlobalsContext {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        writeln!(f, "Globals: ")?;
```

Comment:
When printing the SSA, "Globals:" shows up as the first thing. Is this intended? For example:

```
Globals:
@v0 = Field 1

acir(inline) fn main f0 {
  b0():
    call f1(u1 1, Field 1)
    return
}
```

Just asking because eventually we'd have to parse globals, and maybe not having "Globals:" here could be easier (also less noise). Also this could be changed later on so feel free to ignore this comment.

Reply:
I did add it intentionally but was not sure on it because as you mentioned it adds some noise. I guess you think it is clear on its own we are in the global space without the "Globals:"?

Reply:
Oh, maybe it's fine, we can easily parse that. I was looking at how it's in LLVM, it's like this:

```
@variable = global i32 21
```

In our case that could be:

```
@v0 = global Field 1
```

but I'm not sure I actually like it 😄 

Another alternative is to print "// Globals:" instead of "Globals": the SSA parser already supports comments so that would be ignored.

Reply:
I gated printing globals 
```rust
if self.globals.dfg.values_iter().len() > 0 {
     write!(f, "{}", self.globals)?;
}
```
as to avoid triggering the SSA parser for now. I could switch it to `// Globals: ` and remove that check? I also feel `@v*` states clearly that we have a global as that is the convention we are following.

Reply:
Either way is fine. Thank you!

Reply:
I agree that the leading `Globals: ` or `// Globals:` is probably unnecessary and can be removed

Reply:
I have removed it